blocks.js

```diff
@@ -9584,6 +9705,15 @@ SymbolMorph.prototype.setLabelColor = function (
     this.setColor(textColor);
 };
 
+// SymbolMorph to "scratchblocks"
+
+SymbolMorph.prototype.stringify = function () {
+    var symbol = {
```

Comment:
This format seems confusing to me, especially since there's only one option. 

Why not just use a simple conditional?